extensions/aws2-s3/deployment/src/main/java/org/apache/camel/quarkus/component/aws2/s3/deployment/Aws2S3Processor.java

```diff
@@ -48,17 +46,9 @@ FeatureBuildItem feature() {
         return new FeatureBuildItem(FEATURE);
     }
 
-    @BuildStep
-    ExtensionSslNativeSupportBuildItem activateSslNativeSupport() {
```

Comment:
Shouldn't we produce ExtensionSslNativeSupportBuildItem in AwsCommonsProcessor if we remove it here? I guess the AWS API client communicates over HTTPS, so SSL is needed.

Reply:
Tests are working

Reply:
Btw I'll check again tomorrow